assets/js/phoenix_live_view/dom.js

```diff
@@ -193,6 +194,16 @@ let DOM = {
     }
   },
 
+  syncPendingAttrs(fromEl, toEl){
+    if(!fromEl.hasAttribute(PHX_REF_SRC)){ return }
+    PHX_EVENT_CLASSES.forEach(className => {
+      fromEl.classList.contains(className) && toEl.classList.add(className)
+    })
+    PHX_PENDING_ATTRS.filter(attr => fromEl.hasAttribute(attr)).forEach(attr => {
+      toEl.setAttribute(attr, fromEl.getAttribute(attr))
+    })
```

Comment:
Perf nit: avoid the extra loop by moving the `filter -> &hasAttribute/1` check into the `forEach` as an `if`.